modules/actions/pool/sync-join-exits-v2.ts

```diff
@@ -1,114 +1,92 @@
-// import { Chain, PoolEventType } from '@prisma/client';
-// import { prisma } from '../../../prisma/prisma-client';
-// import type { BalancerSubgraphService } from '../../subgraphs/balancer-subgraph/balancer-subgraph.service';
-// import { JoinExit_OrderBy, OrderDirection } from '../../subgraphs/balancer-subgraph/generated/balancer-subgraph-types';
-// import { roundToHour } from '../../common/time';
+import { Chain, PoolEventType } from '@prisma/client';
+import { prisma } from '../../../prisma/prisma-client';
+import type { BalancerSubgraphService } from '../../subgraphs/balancer-subgraph/balancer-subgraph.service';
+import { JoinExit_OrderBy, OrderDirection } from '../../subgraphs/balancer-subgraph/generated/balancer-subgraph-types';
+import { daysAgo } from '../../common/time';
+import { joinExitsUsd } from '../../sources/enrichers/join-exits-usd';
 
-// /**
-//  * Get the join and exit events from the subgraph and store them in the database
-//  *
-//  * @param vaultSubgraphClient
-//  */
-// export const syncJoinExitsV2 = async (v2SubgraphClient: BalancerSubgraphService, chain: Chain): Promise<string[]> => {
-//     const vaultVersion = 2;
+/**
+ * Get the join and exit events from the subgraph and store them in the database
+ *
+ * @param vaultSubgraphClient
+ */
+export const syncJoinExitsV2 = async (v2SubgraphClient: BalancerSubgraphService, chain: Chain): Promise<string[]> => {
```

Comment:
daysToSync should be passed here

Reply:
do you plan to use a different time per chain? otherwise i can set it as a const here, so it stays in the same context.